app/controllers/TEAHubController.scala

```diff
@@ -0,0 +1,15 @@
+package controllers
+
+import play.api.mvc.Action
+import play.api.mvc._
+import services.TogglService
+import scala.concurrent.{ExecutionContext, Future}
+/**Groups all the action methods*/
+class TEAHubController(togglService: TogglService)(implicit executionContext: ExecutionContext) extends Controller {
```

Comment:
This can be more clear - It is a controller for projects for now. Also missing `@param` for togglService